src/dom/document.ts

```diff
@@ -152,6 +152,14 @@ export class Document extends Node {
   get title(): string {
     return this.querySelector("title")?.textContent || "";
   }
+  set title(value) {
```

Comment:
Can you add explicit `value: string` here? It's required for JSR

Reply:
Done